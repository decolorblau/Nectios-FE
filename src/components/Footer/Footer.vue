<template>
  <v-footer padless class="footer">
    <v-col class="text-center" cols="12">
      <p class="text">Made by deColorBlau</p>
      <p class="text">- NECTIOS -</p>
      <p class="text privacy">Privacy policy | Legal Notice</p>
    </v-col>
  </v-footer>
</template>

<script>
export default {
  name: "Footer",
};
</script>

<style lang="scss" scoped>
.footer {
  background-color: rgb(250, 250, 250);
}
.text {
  color: #999999;
  font-size: 16px;
}
.privacy {
  margin-top: 10px;
  font-size: 14px;
}
</style>
