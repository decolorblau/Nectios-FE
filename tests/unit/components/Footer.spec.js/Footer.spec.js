import Footer from "@/components/Footer/Footer.vue";
import { shallowMount } from "@vue/test-utils";

describe("Given a Login component", () => {
  describe("When is rendered", () => {
    test("Then it matches de snapshot", () => {
      const wrapper = shallowMount(Footer);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
