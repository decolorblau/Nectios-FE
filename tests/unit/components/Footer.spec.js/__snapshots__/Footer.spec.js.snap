// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Login component When is rendered Then it matches de snapshot 1`] = `
<v-footer padless="" class="footer">
  <v-col class="text-center" cols="12">
    <p class="text">Made by deColorBlau</p>
    <p class="text">- NECTIOS -</p>
    <p class="text privacy">Privacy policy | Legal Notice</p>
  </v-col>
</v-footer>
`;
